README.md

```diff
@@ -33,7 +33,7 @@ $ make build
 Using the provider
 ------------------
 
-The provider supports verions `3.8.x` and `3.7.x` of RabbitMQ. It may still work with version`3.6.x`, however this version is no longer supported.
+The provider supports versions `3.10.x`,`3.9.x`,`3.8.x` and `3.7.x` of RabbitMQ. It may still work with version`3.6.x`, however this version is no longer supported.
```

Comment:
>  can we remove them from the list of officially supported versions for this provider ?

I think we can change that into:

```suggestion
The provider supports versions `3.10.x` and `3.9.x` of RabbitMQ. It may still work with versions `3.8.x` and `3.7.x`, however these versions are no longer officialy supported.
```